examples/github-appcards/src/utils/github.ts

```diff
@@ -0,0 +1,296 @@
+const baseUrl = "https://api.github.com";
+const token = import.meta.env.VITE_GH_ACCESS_TOKEN;
+
+const headers = new Headers({
+  Authorization: `token ${token}`,
+  Accept: "application/vnd.github.v3+json",
+});
+
+/**
+ * Fetches a list of projects under a specific username and repository
+ *
+ * Returns {
+ *  name: string
+ *  body: string
+ *  id: number
+ * }
+ */
+export const fetchGitHubProjects = async (username: string, repo: string) => {
+  const gitHubProjects = fetch(
+    `${baseUrl}/repos/${username}/${repo}/projects`,
+    {
+      method: "GET",
+      headers: headers,
+    }
+  )
+    .then((response) => response.json())
+    .then((result) => {
+      const gitHubProjectData = result.map((project: any) => {
+        return {
```

Comment:
Am i missing something or is this map not doing anything? In any case there's no need to store the results in a const first, you could just `return result.map(....)`